core/templates/pages/volunteer-page/volunteer-page.component.ts

```diff
@@ -37,10 +37,29 @@ import { UrlInterpolationService } from
 export class VolunteerPageComponent implements OnInit, OnDestroy {
   directiveSubscriptions = new Subscription();
   mapImgPath = '';
-  art = {};
-  development = {};
-  learnerFeedback = {};
-  lessonCreation = {};
+  art!: {
+    images: string[]; caption: {
+      content: string; name:
```

Comment:
Weird line breaks, and pretty inconsistent throughout this file. Just break after every semicolon perhaps (within the caption dict)?